libi2pd/Crypto.cpp

```diff
@@ -1246,7 +1246,9 @@ namespace crypto
 	void InitCrypto (bool precomputation)
 	{
 		i2p::cpu::Detect ();	
+#if LEGACY_OPENSSL
```

Comment:
how do we initialize in case of 1.1?

Reply:
You don't. 1.1 handles this automatically. See: https://www.openssl.org/news/changelog.html#x12

Reply:
That?

 ```
 *) Added support for auto-initialisation and de-initialisation of the library.
     OpenSSL no longer requires explicit init or deinit routines to be called,
     except in certain circumstances. See the OPENSSL_init_crypto() and
     OPENSSL_init_ssl() man pages for further information.
     [Matt Caswell]
```

Reply:
Yes. OPENSSL_init_ssl only needs to be called in case of non-default options such as initializing the engine.